pkg/cluster/controller.go

```diff
@@ -33,7 +36,8 @@ func runController(ctx context.Context, wg *sync.WaitGroup, dataDir string) erro
 	}
 
 	args := []string{
-		"--bind-address=0.0.0.0:11252",
+		"--bind-address=0.0.0.0",
+		"--secure-port=11252",
```

Comment:
This port is not respected, but the docs show this flag. I'm not sure what the right one is.